### PR TITLE
Style_guide

### PR DESCRIPTION
**Pull request by rahulhazra97**
## Style Guide 

This PR contains all the markdown information for the Style Guide and Layout standards explained in the wiki section of librabrium git hub.

## Review Changes

💻 [Wiki Preview](https://github.com/rahulhazra97/Documentation-Guide/wiki)

🎫 [DOC-766](https://spectrocloud.atlassian.net/browse/DOC-766)

### CODE REVIEW

File: styleguide/spectro-cloud-word-list.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Spectro Cloud Word List
+
```
Comment (karl-cardenas-coding) [inline styleguide/spectro-cloud-word-list.md:2]: Add an intro to this page.

File: styleguide/how-to.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+List out the required software version or hardware the practitioner must have installed and available to enable the feature.
+
+</td>
+</tr> <!-- End Prerequisites row -->
+<tr> <!-- Enablement row -->
+<td> Enablement </td>
+<td>
+
+This section contains the sequential steps required to enable the feature. The steps may be console actions, commands, or scrips the user must invoke. The reader should be able to follow the steps sequentially and successfully enable the feature. As the author, you can change the title to something that makes sense. Just ensure you follow the style guide and its [guidance](https://spectrocloud.atlassian.net/wiki/spaces/DE/pages/1765933057/Spectro+Cloud+Internal+Style+Guide#Headings) for headings. You can use smaller headings in between to help break up the topic. Example: <br />
```
Comment (karl-cardenas-coding) [inline styleguide/how-to.md:75]: Remove the internal confluence link.

File: styleguide/Tutorials.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+ [Markdown Layout](#markdown_layout)
+
+ [Section Breakdown](#section-breakdown)
+
+
+## Tutorial Plan
+
+All tutorials require a tutorial plan document with stakeholder signoff before authoring. Writing the plan out prevents incorrect deliverables and reduces the chance of time spent inefficiently. The tutorial plan process will help you create a much higher-quality tutorial.
+ 
+* To get started, make a copy of the tutorial plan [reference](https://docs.google.com/document/d/1ql_vkaj6dHsobCu0AYuvUXoQ8eojBzetAN3RoGxUepc/edit).
```
Comment (karl-cardenas-coding) [inline styleguide/Tutorials.md:20]: @rahulhazra97 let's use the tutorial plan template file and add it as a child page and link to it. Update the section respectively. 

https://docs.google.com/document/d/1ql_vkaj6dHsobCu0AYuvUXoQ8eojBzetAN3RoGxUepc/edit#

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Feedback and suggestions below. Two main things @rahulhazra97 

- Update the file names to all be lowercase.
- All URLs must be updated to use the librirarum wiki URLs instead of your personal repo.

File: styleguide/Home.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+   
+
+Also, check the [Content Layout](https://github.com/rahulhazra97/Documentation-Guide/wiki/Content-Layout) page listing resources to structure your content layout: 
+* [How-to Layout](https://github.com/rahulhazra97/Documentation-Guide/wiki/How-to-Layout) - A how-to’s purpose is to help the user perform a particular task correctly. Refer to the [How-to Layout](https://github.com/rahulhazra97/Documentation-Guide/wiki/How-to-Layout) to use the layout and learn how to create impactful How-to documentation.
+
+* [Packs Layout](https://github.com/rahulhazra97/Documentation-Guide/wiki/Packs-Layout) - The Packs Layout provides an overview and guidelines for organizing and structuring packs within the documentation. A pack refers to a collection of related components, features, or modules that are grouped together for a specific purpose or functionality. Refer to the [Packs Layout][Packs Layout](https://github.com/rahulhazra97/Documentation-Guide/wiki/Packs-Layout) to use the packs layout.
```
Comment (rahulhazra97) [inline styleguide/Home.md:11]:
```suggestion
* [Packs Layout](https://github.com/rahulhazra97/Documentation-Guide/wiki/Packs-Layout) - The Packs Layout provides an overview and guidelines for organizing and structuring packs within the documentation. A pack refers to a collection of related components, features, or modules that are grouped together for a specific purpose or functionality. Refer to the [Packs Layout](https://github.com/rahulhazra97/Documentation-Guide/wiki/Packs-Layout) to use the packs layout.
```

Comment (rahulhazra97) [inline styleguide/Home.md:11]:
```suggestion
* [Packs Layout](https://github.com/rahulhazra97/Documentation-Guide/wiki/Packs-Layout) - The Packs Layout provides an overview and guidelines for organizing and structuring packs within the documentation. A pack refers to a collection of related components, features, or modules that are grouped together for a specific purpose or functionality. Refer to the [Packs Layout](https://github.com/rahulhazra97/Documentation-Guide/wiki/Packs-Layout) to use the packs layout.
```

File: styleguide/spectro-cloud-style-guide.md
Comment (karl-cardenas-coding) [inline styleguide/spectro-cloud-style-guide.md:1]: @rahulhazra97 Go through my original feedback for this document. Several of my suggestion and things I called out are missing.

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]

File: styleguide/spectro-cloud-style-guide.md
Comment (rahulhazra97) [reply styleguide/spectro-cloud-style-guide.md:1]: I have created a list of the things you have asked me to finish for this file. I made a note yesterday while working on this. I'll share this with you when we sync-up

File: styleguide/spectro-cloud-word-list.md (hunk 1 above)
Comment (rahulhazra97) [reply styleguide/spectro-cloud-word-list.md:2]: Added from line 3.

File: styleguide/tutorial-layout.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+When you are ready for a review, tag the education team or notify the team by dropping a message in the Slack `education` channel.
+
+### Tutorial Plan Reference
+To get started with your tutorial plan, create a copy of the following template: 
+
+
+## Tutorial Layout 
+
+All Spectro Cloud tutorials are expected to have the same look and feel. The tutorial text and voice must adhere to the Spectro Cloud[ style guide](https://github.com/rahulhazra97/Documentation-Guide/wiki/Spectro-Cloud-Style-Guide) for a universal look and feel. The tutorial layout must also feel familiar to the reader; therefore, all tutorials must follow the layout standard described on this page.
```
Comment (rahulhazra97) [inline styleguide/tutorial-layout.md:38]:
```suggestion
All Spectro Cloud tutorials are expected to have the same look and feel. The tutorial text and voice must adhere to the Spectro Cloud [style guide](https://github.com/rahulhazra97/Documentation-Guide/wiki/Spectro-Cloud-Style-Guide) for a universal look and feel. The tutorial layout must also feel familiar to the reader; therefore, all tutorials must follow the layout standard described on this page.
```

File: styleguide/tutorial-layout.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+| [Concept]                 | The title of one of the main concepts of the tutorial. This could be one of several main concepts.  Example - “Configure the Environment” |
+| [Sub-Concept]             |  The title of a child concept stems from the parent concept.  Example - “Create Secondary Cluster” OR “App Model  Overview” |
+| Cleanup                   |  All tutorials should ideally be deployed through infrastructure as code (IaC) if applicable.  All resources deployed by the tutorial should be removed. It is our responsibility to help and provide guidance to the user on how to remove all resources. Sometimes this section is brief with a simple command `terraform destroy -auto-approve`.  |
+| Wrap-Up                   |  This section summarizes the key learning concepts and the actions the practitioner conducted. Additionally, this section should link to or suggest the next set of topics that the practitioner can dive into. |
+
+For questions or concerns, please ask in the  Slack channel `education`.
```
Comment (rahulhazra97) [inline styleguide/tutorial-layout.md:90]:
```suggestion
For questions or concerns, please mail to our email address [spectro-docs@spectrocloud.com](spectro-docs@spectrocloud.com).
```

[the remark this answers is not in the document]
Comment (rahulhazra97) [reply styleguide/tutorial-layout.md:90]: Done.

File: styleguide/tutorial-layout.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+
+ ![image](https://github.com/rahulhazra97/Documentation-Guide/assets/126905240/72e6eca4-992a-45ea-9c80-ab5f3708deaf)
+
+* Next, copy the template to a Google Drive location of your choosing. <br /> <br />
+![image](https://github.com/rahulhazra97/Documentation-Guide/assets/126905240/1d2ce48e-ca26-47e5-be2c-4e4774531783)
+
+When you are ready for a review, tag the education team or notify the team by dropping a message in the Slack `education` channel.
+
+### Tutorial Plan Reference
+To get started with your tutorial plan, create a copy of the following template:
```
[the remark this answers is not in the document]
Comment (rahulhazra97) [reply styleguide/tutorial-layout.md:33]: I did create a child page as reference template on line 20. Is that not a child page? 

Rest removed and correct to our email address.

File: styleguide/Home.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+* [How-to Layout](https://github.com/rahulhazra97/Documentation-Guide/wiki/How-to-Layout) - A how-to’s purpose is to help the user perform a particular task correctly. Refer to the [How-to Layout](https://github.com/rahulhazra97/Documentation-Guide/wiki/How-to-Layout) to use the layout and learn how to structure a how-to guide.
+
+* [Packs Layout](https://github.com/rahulhazra97/Documentation-Guide/wiki/Packs-Layout) - The Packs Layout provides an overview and guidelines for how to organize and structure a pack documentation page. Refer to the [Packs Layout](https://github.com/rahulhazra97/Documentation-Guide/wiki/Packs-Layout) guide to learn more.
+  
+* [Troubleshooting Layout](https://github.com/rahulhazra97/Documentation-Guide/wiki/Troubleshooting-Layout) - All Spectro Cloud troubleshooting guides are expected to have the same look and feel. The guide must also feel familiar to the reader. Therefore, all troubleshooting guides must follow the layout standard described. Refer to the [Troubleshooting Layout](https://github.com/rahulhazra97/Documentation-Guide/wiki/Troubleshooting-Layout) to use the troubleshooting layout.
```
Comment (karl-cardenas-coding) [inline styleguide/Home.md:15]:
```suggestion
* [Troubleshooting Layout](https://github.com/rahulhazra97/Documentation-Guide/wiki/Troubleshooting-Layout) - All Spectro Cloud troubleshooting guides are expected to have the same look and feel. The guide must also feel familiar to the reader. Refer to the [Troubleshooting Layout](https://github.com/rahulhazra97/Documentation-Guide/wiki/Troubleshooting-Layout) to learn how to structure a troubleshooting guide..
```

File: styleguide/tutorial-layout.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
+# Tutorial Layout
+
+To create an impactful tutorial guide, you must create a tutorial plan and obtain stakeholder approval for alignment. Adhere to the [Tutorial Plan](#tutorial-plan) section, and the [Tutorial Layout](#tutorial-layout) section to deliver accurate, user-friendly, and impactful tutorials consistently.
```
Comment (karl-cardenas-coding) [inline styleguide/tutorial-layout.md:3]:
```suggestion
All Spectro Cloud tutorials must adhere to the following layout. Review the layout guide when structuring a tutorial. Before you bring writing a tutorial, we recommend you go through the exercise of planning out the tutorial. Use the [Tutorial Plan Template](https://github.com/rahulhazra97/Documentation-Guide/wiki/Tutorial-Plan-Template) to get started. Planning out a tutorial and identifying the scope and audience results in a higher quality deliverable and prevents writing content that may not be applicable to the tutorial. 

Reach out to us at spectro-docs@spectrocloud.com to propose a tutorial idea and share the plan with us.
```

We can combine the intro with the following section.

File: styleguide/tutorial-layout.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+| [Concept]                 | The title of one of the main concepts of the tutorial. This could be one of several main concepts.  Example - “Configure the Environment” |
+| [Sub-Concept]             |  The title of a child concept stems from the parent concept.  Example - “Create Secondary Cluster” OR “App Model  Overview” |
+| Cleanup                   |  All tutorials should ideally be deployed through infrastructure as code (IaC) if applicable.  All resources deployed by the tutorial should be removed. It is our responsibility to help and provide guidance to the user on how to remove all resources. Sometimes this section is brief with a simple command `terraform destroy -auto-approve`.  |
+| Wrap-Up                   |  This section summarizes the key learning concepts and the actions the practitioner conducted. Additionally, this section should link to or suggest the next set of topics that the practitioner can dive into. |
+
+For any further information contact : `spectro-docs@spectrocloud.com
```
Comment (karl-cardenas-coding) [inline styleguide/tutorial-layout.md:77]:
```suggestion
For any further information contact : spectro-docs@spectrocloud.com
```

File: styleguide/spectro-cloud-word-list.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+The Spectro Cloud Word List is a reference to ensure we speak the same language.  It provides unified language and terminology across our communication materials, including documentation, marketing content, and internal communications. 
+
+We believe in clear and consistent communication and use this reference to reflect a unified voice and consistent messaging.  Review this list to avoid confusion, misinterpretation, or inconsistency when discussing key concepts and ideas.
```
Comment (karl-cardenas-coding) [inline styleguide/spectro-cloud-word-list.md:5]:
```suggestion
The Spectro Cloud Word List is a reference to ensure we speak the same language.  The following list is intended to help you understand the word choices we have made regarding terms commonly used in our documentation.
```

File: styleguide/spectro-cloud-word-list.md [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+
+- log in - use as a verb. 
+
+- login - (noun, adj).
+
+- Main menu - the main navigation pane of the product.
+
+- Palette Virtual Cluster - may be referenced as “virtual cluster” after initially defined in its full name.
+
+- prerequisite(s) - no hyphen.
```
Comment (karl-cardenas-coding) [inline styleguide/spectro-cloud-word-list.md:29]:
```suggestion
- prerequisites - no hyphen.
```

File: styleguide/troubleshooting-layout.md [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+
+## Section Breakdown
+
+| Section        | Description |
+|----------------|-------------|
+| Scenario (Title)   | The name assigned to the troubleshooting scenario. Aim for a brief title, ideally less than five words. |
+| Intro Paragraph  | The introduction sets the context for the troubleshooting guide and explains the issue. |
+| Debugging Steps  |  The step-by-step flow of how to address the issue or how to identify the problem. |
+
+For questions or concerns, please ask in the  Slack channel `education`.
```
Comment (karl-cardenas-coding) [inline styleguide/troubleshooting-layout.md:33]:
```suggestion
For questions or concerns, email us at spectro-docs@spectrocloud.com. 
```

File: styleguide/Home.md [hunk 14]
```diff
@@ -0,0 +1,22 @@
...
+
+
+
+* [Tutorial Layout](https://github.com/rahulhazra97/Documentation-Guide/wiki/Tutorial-Layout) - Our tutorials are written and structured in a specific manner and style.  Refer to the [Tutorial Layout](https://github.com/rahulhazra97/Documentation-Guide/wiki/Tutorial-Layout) guide to learn how to structure a tutorial for Spectro Cloud documentation.
+
+Refer to the [Spectro Cloud Word List](https://github.com/rahulhazra97/Documentation-Guide/wiki/Spectro-Cloud-Word-List) to learn more about the choices we have made when it comes to certain words and their usage in our writing. 
+
```
Comment (karl-cardenas-coding) [inline styleguide/Home.md:22]:
```suggestion

For questions or concerns, email us at spectro-docs@spectrocloud.com
```

Let's add a way for users to contact us at the landing page.

File: styleguide/packs-layout.md [hunk 15]
```diff
@@ -0,0 +1,146 @@
...
+
+:warning: If no troubleshooting steps are available then omit this section. <br />  <br /> This section contains troubleshooting steps pertaining to the respective Pack.
+
+</td>
+</tr> <!-- End Troubleshooting row -->
+<tr> <!-- Terraform row -->
+<td> Terraform </td>
+<td>
+
+^ Extra blank line above!
```
Comment (karl-cardenas-coding) [inline styleguide/packs-layout.md:115]:
```suggestion
```
Not sure what this is but let's remove it.

File: styleguide/how-to.md [hunk 16]
```diff
@@ -0,0 +1,103 @@
+# How-To-Guide
```
Comment (karl-cardenas-coding) [inline styleguide/how-to.md:1]:
```suggestion
# How-to Layout
```

**Review by karl-cardenas-coding — APPROVED** [review]
@rahulhazra97 this looks loos good. Besides the handful of suggestion I provided in this final review. Do the following things before your merge:

1. Fix the duplicate titles that is occurring in all layout pages, including the content layout page (see screenshot).

2. Fix all the URLs to make sure they don't point to your repo. You may have to add this to our actual repo as a wiki to get the real URLs.

CleanShot 2023-06-01 at 14 23 07

**Pull request merged**